subscriber/common/database/ares_database.go

```diff
@@ -0,0 +1,76 @@
+package database
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/uber-go/tally"
+	"github.com/uber/aresdb/client"
+	"github.com/uber/aresdb/subscriber/config"
+	"github.com/uber/aresdb/utils"
+	"go.uber.org/zap"
+)
+
+// AresDatabase is an implementation of Database interface for saving data to ares
+type AresDatabase struct {
+	ServiceConfig config.ServiceConfig
+	Scope         tally.Scope
+	ClusterName   string
```

Comment:
it'd be great if we can get rid of ClusterName and use InstanceName by now..

Reply:
need sync up distribution design later